paymentsheet/src/test/java/com/stripe/android/customersheet/state/DefaultCustomerSheetLoaderTest.kt

```diff
@@ -594,16 +601,11 @@ class DefaultCustomerSheetLoaderTest {
     }
```

Comment:
`CustomerSheetHacks` now creates a real `ElementsSessionRepository` instance so we instead just mimic that behavior with `initializationDataSourceProvider`.

This is fine as it keeps these unit tests isolated to the function of `CustomerAdapterDataSource`. Initializing of `CustomerSheetHacks` is covered by our integrated `CustomerSheet` tests.

Reply:
Tested this against `ShampooRule`